.github/workflows/build_all_apps.yml

```diff
@@ -0,0 +1,86 @@
+name: Build all Rust apps
+
+on:
+    workflow_dispatch:
+        inputs:
+            sdk_branch:
+                type: string
+                required: false
+                default: ''
+    pull_request: 
+
+
+jobs:    
+    test-build:
+        name: Build for all targets
+        strategy:
+            fail-fast: false
+            matrix:
+                include: 
+                    - repo: 'app-mobilecoin'
+                      branch: 'develop'
+                      build_path: './fw'
```

Comment:
Can we parse the `ledger_app.toml` manifest instead?

Reply:
WIP